src/features/call/layouts/CallLayout.tsx

```diff
@@ -1,113 +1,87 @@
-  );
-  const { deleteCall } = useCallMutations(assignment!.organization.id);
-
   return (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: 'column',
-      }}
-    >
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'space-between',
-          p: 2,
-        }}
-      >
-        {isDetailsPage && (
-          <Link href="/my/home" passHref>
-            <Button variant="outlined">
-              <Msg id={messageIds.nav.backToHome} />
-            </Button>
-          </Link>
-        )}
-        {isPreparePage && (
-          <Link href="/my/home" passHref>
-            <Button
-              onClick={() => {
-                if (call) {
-                  deleteCall(call.id);
-                }
-              }}
-              variant="outlined"
-            >
-              Stop calling
-            </Button>
-          </Link>
-        )}
-        <Box>
-          {isPreparePage && call && assignment && (
-            <SkipCallDialog
-              assignment={assignment}
-              callId={call?.id}
-              targetName={
-                call?.target.first_name + ' ' + call?.target.last_name
-              }
-            />
-          )}
-          {isDetailsPage && (
-            <Button
-              onClick={() => {
-                router.push(`/call/${callAssId}/prepare`);
-                allocateCall();
-              }}
-              variant="contained"
-            >
-              <Msg id={messageIds.nav.startCalling} />
-            </Button>
-          )}
-          {isPreparePage && <Button variant="contained">Start Call</Button>}
+    <Box>
+      <Box sx={(theme) => ({ backgroundColor: theme.palette.common.white })}>
+        <Box
+          pt={2}
+          sx={{
+            overflow: 'hidden',
+            pl: { sm: 3, xs: 2 },
+            pr: 2,
+            textOverflow: 'ellipsis',
+          }}
+        >
+          <ZUIText noWrap variant="headingMd">
+            {assignment?.title || 'Untitled call assignment'}
+          </ZUIText>
+        </Box>
+
+        <Box sx={{ px: { sm: 3, xs: 2 }, py: 2 }}>
+          <Box
+            sx={{
+              alignItems: 'center',
+              display: 'grid',
+              gap: 1,
+              gridTemplateColumns: '1fr auto',
+              width: '100%',
+            }}
+          >
+            <Box alignItems="center" display="flex" minWidth={0}>
+              {assignment && (
+                <Box alignItems="center" display="flex" sx={{ flexShrink: 0 }}>
+                  <ZUIOrgAvatar
+                    orgId={assignment.organization.id}
+                    title={assignment.organization.title}
+                  />
+                </Box>
+              )}
+
+              <Box
+                maxWidth="100%"
+                minWidth={0}
+                ml={1}
+                sx={{
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
+                }}
```

Comment:
are these two still needed when the `noWrap` property is on the ZUIText? The intention for the `noWrap` property is that it should take care of everything needed to truncate the text

Reply:
True, it works without them! I thought we needed them, I'll remove them in both places! Thanks!!